docs/stackscli/runtime_config.mdx

```diff
@@ -0,0 +1,373 @@
+# Runtime Configuration
+
+Configuration for the CLI can be performed in several ways:
+
+  - On the command line
+
+  - Using environment variables
+
+  - Using a configuration file (YAML or JSON)
+
+When using a configuration file it is possible to setup more than one project.
+
+## Settings
+
+The following tables show the configuration options that are available at the root level and the command level.
+
+### Global Options
+
+The following options can be applied to any command within the Stacks CLI.
+
+| Option                     | Environment Variable               | Description                                                                   | Default               | Options                                      |
+| -------------------------- | ---------------------------------- | ----------------------------------------------------------------------------- | --------------------- | -------------------------------------------- |
+| `-c`, `--config`           | AMIDOSTACKS\_CONFIG                | Path to the configuration file to use                                         |                       |                                              |
+| `-l`, `--loglevel`         | AMIDOSTACKS\_LOG\_LEVEL            | Level of logging to be used                                                   | `info`                | `debug`, `info`, `warning`, `error`, `fatal` |
+| `-f`, `--logformat`        | AMIDOSTACKS\_LOG\_FORMAT           | Output type of the logging                                                    | `text`                | `text`, `json`                               |
+| `--logcolur`               | AMIDOSTACKS\_LOG\_COLOUR           | State if logging should include colour                                        |                       |                                              |
```

Comment:
Potential typo here, it should be `--logcolour`, right?

Reply:
Corrected the typo